packages/fpdart/lib/src/option.dart

```diff
@@ -1,617 +1,159 @@
-import 'either.dart';
-import 'extension/option_extension.dart';
-import 'function.dart';
-import 'io_option.dart';
-import 'task_option.dart';
-import 'typeclass/applicative.dart';
-import 'typeclass/eq.dart';
-import 'typeclass/extend.dart';
-import 'typeclass/filterable.dart';
-import 'typeclass/functor.dart';
-import 'typeclass/hkt.dart';
-import 'typeclass/monad.dart';
-import 'typeclass/monoid.dart';
-import 'typeclass/order.dart';
-import 'typeclass/semigroup.dart';
-
-/// Return a `Some(t)`.
-///
-/// Shortcut for `Option.of(r)`.
-Option<T> some<T>(T t) => Some(t);
-
-/// Return a [None].
-///
-/// Shortcut for `Option.none()`.
-Option<T> none<T>() => const Option.none();
-
-/// Return [None] if `t` is `null`, [Some] otherwise.
-///
-/// Same as initializing `Option.fromNullable(t)`.
-Option<T> optionOf<T>(T? t) => Option.fromNullable(t);
-
-/// Return [Some] of `value` when `predicate` applied to `value` returns `true`,
-/// [None] otherwise.
-///
-/// Same as initializing `Option.fromPredicate(value, predicate)`.
-Option<T> option<T>(T value, bool Function(T) predicate) =>
-    Option.fromPredicate(value, predicate);
-
-final class _OptionThrow {
-  const _OptionThrow();
-}
+part of "effect.dart";
```

Comment:
~~Would a micro module be possible, like [Effect's](https://effect.website/docs/other/micro), but even lighter?
In my head, you could keep `IEffect` as base, but keep `Option`, `Either`, etc from directly interacting with `Effect`, but only `IEffect`. That would make migrating from v1 and integrating it into existing projects easier, by not requiring users to use the Effect system (it's obviously recommended to use the Effect portion), and comes with the side-effect of a (slightly) smaller learning curve, as you can use it in chunks.~~

I thought there were some APIs here that called Effect. Doesn't look like anything really, so I think it should be documented that you can still use `Option`, `Either`, etc, without needing to adopt the Effect system.